src/cascadia/TerminalApp/ContentManager.cpp

```diff
@@ -31,14 +31,14 @@ namespace winrt::TerminalApp::implementation
                                                     TerminalConnection::ITerminalConnection connection)
     {
         auto content = ControlInteractivity{ settings, unfocusedAppearance, connection };
-        content.Closed({ this, &ContentManager::_closedHandler });
-        _content.Insert(content.Id(), content);
+        content.Closed({ get_weak(), &ContentManager::_closedHandler });
+        _content.insert(std::make_pair(content.Id(), content));
         return content;
     }
 
     ControlInteractivity ContentManager::LookupCore(winrt::guid id)
     {
-        return _content.TryLookup(id);
+        return _content.at(id);

```

Comment:
`at` and `TryLookup` aren't the same. `at` will throw an exception if the element can't be found. If you don't want this (for instance if you want to handle invalid `id`s elsewhere), you could either use `_content[id]` (if you expect `LookupCore` to be called with valid `id`s almost all of the time), or use this otherwise:
```cpp
const auto it = _content.find(id);
return it != _content.end() ? *it : ControlInteractivity{ nullptr };
```